From 1878a1c5869b72160fa9331347e9b2a0f0d5bc60 Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Fri, 31 Aug 2012 18:16:28 +0800
Subject: [PATCH 054/304] ENGR00222202 imx: change model name align with
 board.

align the model name with board, use a read model name for
each board.

Signed-off-by: Zhang Jiejing <jiejing.zhang@freescale.com>
---
 arm2_6sl/BoardConfig.mk                | 1 +
 imx53_smd/BoardConfig.mk               | 2 ++
 imx5x/imx5x.mk                         | 1 -
 imx6/arm2/BoardConfigArm2.mk           | 1 +
 imx6/imx6.mk                           | 5 ++---
 imx6/sabresd/SabreSDBoardConfigComm.mk | 1 +
 sabreauto_6q/BoardConfig.mk            | 1 +
 7 files changed, 8 insertions(+), 4 deletions(-)

diff --git a/arm2_6sl/BoardConfig.mk b/arm2_6sl/BoardConfig.mk
index 93956bb..e2ae13b 100755
--- a/arm2_6sl/BoardConfig.mk
+++ b/arm2_6sl/BoardConfig.mk
@@ -8,6 +8,7 @@ include device/fsl-proprietary/gpu-viv/fsl-gpu.mk
 
 USE_CAMERA_STUB := true
 BOARD_HAVE_IMX_CAMERA := false
+PRODUCT_MODEL := ARM2-MX6SL
 
 BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init androidboot.console=ttymxc0
 TARGET_BOOTLOADER_CONFIG := mx6sl:mx6sl_arm2_android_config
diff --git a/imx53_smd/BoardConfig.mk b/imx53_smd/BoardConfig.mk
index d0ba255..d137515 100755
--- a/imx53_smd/BoardConfig.mk
+++ b/imx53_smd/BoardConfig.mk
@@ -9,6 +9,8 @@ TARGET_BOOTLOADER_BOARD_NAME := SMD
 BOARD_SOC_CLASS := IMX5X
 BOARD_SOC_TYPE := IMX53
 
+PRODUCT_MODEL := SMD-MX53
+
 # Wifi
 BOARD_WLAN_VENDOR := ATHEROS
 BOARD_WLAN_DEVICE := ar6003
diff --git a/imx5x/imx5x.mk b/imx5x/imx5x.mk
index 060f265..39b35f1 100644
--- a/imx5x/imx5x.mk
+++ b/imx5x/imx5x.mk
@@ -5,7 +5,6 @@ $(call inherit-product, frameworks/base/data/sounds/AllAudio.mk)
 # overrides
 PRODUCT_BRAND := freescale
 PRODUCT_MANUFACTURER := freescale
-PRODUCT_MODEL := ICS AOSP on imx5x
 
 # Android infrastructures
 PRODUCT_PACKAGES += \
diff --git a/imx6/arm2/BoardConfigArm2.mk b/imx6/arm2/BoardConfigArm2.mk
index bdb4ec1..de52991 100644
--- a/imx6/arm2/BoardConfigArm2.mk
+++ b/imx6/arm2/BoardConfigArm2.mk
@@ -7,6 +7,7 @@ include device/fsl/imx6/BoardConfigCommon.mk
 TARGET_BOOTLOADER_BOARD_NAME := ARM2
 
 BOARD_SOC_CLASS := IMX6
+PRODUCT_MODEL := ARM2-MX6DQ
 
 # Wifi
 BOARD_WLAN_VENDOR 			 := ATHEROS
diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index f31fd43..60ea042 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -3,9 +3,8 @@ $(call inherit-product, $(SRC_TARGET_DIR)/product/generic.mk)
 $(call inherit-product, $(TOPDIR)frameworks/base/data/sounds/AllAudio.mk)
 
 # overrides
-PRODUCT_BRAND := freescale
-PRODUCT_MANUFACTUER := freescale
-PRODUCT_MODEL := ICS AOSP on imx6q
+PRODUCT_BRAND := Freescale
+PRODUCT_MANUFACTURER := Freescale
 
 # Android infrastructures
 PRODUCT_PACKAGES += \
diff --git a/imx6/sabresd/SabreSDBoardConfigComm.mk b/imx6/sabresd/SabreSDBoardConfigComm.mk
index fa01935..03d8ce5 100755
--- a/imx6/sabresd/SabreSDBoardConfigComm.mk
+++ b/imx6/sabresd/SabreSDBoardConfigComm.mk
@@ -6,6 +6,7 @@
 include device/fsl/imx6/BoardConfigCommon.mk
 
 TARGET_BOOTLOADER_BOARD_NAME := SABRESD
+PRODUCT_MODEL := SABRESD-MX6DQ
 
 # Wifi
 BOARD_WLAN_VENDOR 			 := ATHEROS
diff --git a/sabreauto_6q/BoardConfig.mk b/sabreauto_6q/BoardConfig.mk
index efa151f..edef186 100644
--- a/sabreauto_6q/BoardConfig.mk
+++ b/sabreauto_6q/BoardConfig.mk
@@ -8,6 +8,7 @@ TARGET_BOOTLOADER_BOARD_NAME := SABREAUTO
 
 BOARD_SOC_CLASS := IMX6
 BOARD_SOC_TYPE := IMX6Q
+PRODUCT_MODEL := SABREAUTO-MX6Q
 
 USE_OPENGL_RENDERER := true
 TARGET_CPU_SMP := true
-- 
1.8.0

